python/samples/getting_started/00-getting-started.ipynb

```diff
@@ -20,6 +20,72 @@
     "%pip install semantic-kernel==1.1.0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Config\n",
+    "\n",
+    "Let's get started with the necessary configuration to run Semantic Kernel. For Notebooks, we require a `.env` file with the proper settings for the model you use. You may copy the `.env.example` file from this directory and place it either in this current directory, or in the main semantic-kernel/python directory.\n",
```

Comment:
Is there a benefit to either choice for the location of the .env file? If not, maybe we should just pick one for these instructions. Less choices to make when getting started is better.

Reply:
Good callout. I'd say we recommend the user to place it in the semantic-kernel/python root as a standard. Can be easier to find then having to dig deeper into a samples directory, and also can then be used for later development. Will update the text.